lib/system_win32.c

```diff
@@ -367,6 +367,7 @@ bool Curl_verify_windows_version(const unsigned int majorVersion,
  */
 HMODULE Curl_load_library(LPCTSTR filename)
 {
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Comment:
We have `CURL_WINDOWS_APP` for that, which also checks if this check works and includes the necessary header if it can be included.

Reply:
Thanks, I'll fixup and force-push!